content/millennium/r4/other/health-cards.md

```diff
@@ -65,6 +66,34 @@ _Authorization Types_
 
 The OAuth2 token must include `Patient.read` and `Immunization.read` scopes.
 
+### Laboratory Test Result Credentials (DM)
+
+Represents the patient and the clinical data related to a laboratory test result
+
+Contains the following resources:
+
+* A single Patient resource that follows the [VaccinationCredentialPatientDM](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-dm.html#tab-ms) profile with the following fields, if valued:
+  * [Patient official name](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-definitions.html#Patient.name){:target="_blank"}
+  * [Date of Birth](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-vaccination-credential-patient-definitions.html#Patient.birthDate){:target="_blank"}
+
+* One or more Observation resources that follow the [Covid19LaboratoryResultObservationDM](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-covid19-laboratory-result-observation-dm.html) profile with the following fields, if valued:
+  * [Status](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-covid19-laboratory-result-observation-definitions.html#Observation.status){:target="_blank"}
+  * [Code](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-covid19-laboratory-result-observation-definitions.html#Observation.code){:target="_blank"}
+  * [Subject](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-covid19-laboratory-result-observation-definitions.html#Observation.subject){:target="_blank"}
+  * [Effective date time](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-covid19-laboratory-result-observation-definitions.html#Observation.effectiveDateTime){:target="_blank"}
+  * [Value (one of)](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-covid19-laboratory-result-observation-definitions.html#Observation.value[x]){:target="_blank"}
+    * [Value codeable concept](http://build.fhir.org/ig/dvci/vaccine-credential-ig/branches/main/StructureDefinition-covid19-laboratory-result-observation-definitions.html#Observation.value[x]:valueCodeableConcept){:target="_blank"}
```

Comment:
@Mohammad-Isyroqi-Fathan not sure if it is just me but when I click the value[x] link it takes me to the top of the page instead of directly to value[x]

Reply:
They are taking me to each corresponding field for me. How did you try to access them?

Reply:
by right clicking the link and open in new tab.

Reply:
never mind - user error. I wasn't in the view file mode. the links work correctly.